lib/assertions.js

```diff
@@ -444,6 +444,24 @@ module.exports = function (expect) {
         if (value === '') {
             throw new Error("The '" + expect.testDescription + "' assertion does not support a prefix of the empty string");
         }
+        var isTruncated;
+        var outputSubject;
+        var contextLength = value.length + 25;
+        if (subject.length > contextLength) {
+            var truncationIndex = subject.indexOf(' ', value.length + 1);
+            if (truncationIndex > -1 && truncationIndex < contextLength) {
+                outputSubject = subject.substring(0, truncationIndex);
+            } else {
+                outputSubject = subject.substring(0, contextLength);
+            }
+            isTruncated = true;
+        } else {
+            outputSubject = subject;
+            isTruncated = false;
+        }
+        expect.subjectOutput = function (output) {
+            output.jsString("'" + outputSubject.replace(/\n/g, '\\n') + "'" + (isTruncated ? '...' : ''));
```

Comment:
Use jsComment() to render ellipsis addressed in e76d315.

Looking into singleQuotedString().